assets/style.css

```diff
@@ -0,0 +1,505 @@
+  padding: 3%;
+  display: grid;
+  grid-template-columns: 1fr 1fr 3fr;
+  color: white;
+  justify-content: center;
+  align-items: center;
+}
+
+.mainProgramWrapper h3 {
+  color: #ff5036;
+}
+
+.programsContainer {
+  display: grid;
+  gap: 8px;
+  padding-top: 2%;
+}
+
+.holeProgram {
+  background-color: #ff4f38;
+  padding: 5%;
+  margin: 5%;
+  color: white;
+  text-align: center;
+}
+
+.holeProgram a {
+  text-decoration: none;
+  color: white;
+}
+
+/* Featured lecturers */
+.lecturers > h1 {
+  text-align: center;
+  margin-top: 5%;
+}
+
+.line {
+  border: 2px solid #ff4f38;
+  width: 10%;
+  margin: 0 auto;
+}
+
+.lecturersContainer {
+  padding: 5%;
+  display: grid;
+  gap: 20px;
+}
+
+.lecturersWrapper {
+  display: grid;
+  grid-template-columns: 1fr 3fr;
+  gap: 5%;
+}
+
+.lecturersWrapper img {
+  margin-top: 10%;
+  width: 100%;
+}
+
+.lecturersDescription {
+  text-align: left;
+}
+
+.line2 {
+  border: 1px solid #383434;
+  width: 10%;
+  margin-top: 5%;
+  margin-bottom: 5%;
+}
+
+/* partners */
+.partners {
+  background-color: #414246;
+}
+
+.partnersTitle {
+  text-align: center;
+  color: white;
+  padding-top: 5%;
+}
+
+.partners .line3 {
+  border: 2px solid #ff4f38;
+  width: 7%;
+  margin: 0 auto;
+}
+
+.sponsors img {
+  width: 30%;
+  margin: 5%;
+}
+
+.sponsors {
+  padding: 10%;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+/* footer */
+.footer {
+  height: 200px;
+  color: black;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.footerWrapper {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 10%;
+  width: 100%;
+}
+
+.footerWrapper div {
+  text-align: center;
+}
+
+/* about page */
+.aboutIntroDescription {
+  background-color: #fff;
+  border: 1px solid #9b9797;
+}
+
+.aboutBlur {
+  text-align: center;
+}
+
+.introFooterAbout {
+  font-size: small;
+}
+
+/* about logo */
+.aboutLogo {
+  padding: 7%;
+}
+
+.aboutLogoTitle {
+  text-align: center;
+  font-size: x-large;
+}
+
+.aboutLogoDescription {
+  text-align: center;
+  font-size: large;
+  margin-top: 3%;
+}
+
+.aboutLogoImg {
+  display: flex;
+  justify-content: center;
+  margin-top: 5%;
+}
+
+.aboutLogoImg img {
+  border-radius: 40px;
+  width: 80%;
+  text-align: center;
+  margin: 0 auto;
+}
+
+/* previous activities about */
+
+.previousActivities {
+  padding: 5%;
+}
+
+.previousActivitiesTitle {
+  font-size: x-large;
+  text-align: center;
+}
+
+.graduate1 {
+  background-image: url('./images/graduate1.jpg');
+}
+
+.graduate2 {
+  background-image: url('./images/graduate2.jpg');
+}
+
+.graduate {
+  margin-top: 7%;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  height: 200px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  font-size: large;
+  gap: 5px;
+}
+
+/* mobile menu section */
+@media only screen and (max-width: 768px) {
```

Comment:
- [ ] Good job with the design for your mobile version :+1:, but as a project requirement and to improve the mobile user experience you should make sure that the CSS styling is built as **mobile-first**, it means media queries are used to change the design from mobile to desktop, not the other way around.

Reply:
@ghreza-crypto as previously requested kindly ensure you have media queries for only desktop view, this will make your project fulfill project requirement of being mobile first .